demo.py

```diff
@@ -27,7 +20,8 @@
 args = parser.parse_args()
 
 # Load model
-p_wct = PhotoWCT(args)
+p_wct = PhotoWCT()
+p_wct.load_state_dict(torch.load(args.model))
```

Comment:
For ease of use, use a try/except here with the exception message reminding the user to pull the submodule.

Reply:
Thanks. Updated.